addons/im_livechat/models/im_livechat_channel.py

```diff
@@ -40,6 +42,13 @@ def _default_default_message(self):
     title_color = fields.Char(default="#FFFFFF", help="Default title color of the channel once open")
     button_background_color = fields.Char(default="#875A7B", help="Default background color of the Livechat button")
     button_text_color = fields.Char(default="#FFFFFF", help="Default text color of the Livechat button")
+    concurent_session = fields.Boolean()
+    concurent_session_number = fields.Integer(default=10)
+
+    @api.constrains("concurent_session_number", "concurent_session")
+    def _check_concurent_session_number(self):
+        if any(lc.concurent_session and lc.concurent_session_number <= 0 for lc in self):
+            raise ValidationError(_("Concurent session number cannot be 0 or less."))
```

Comment:
It would actually be better to use an SQL constraint. Also, I would put it *after* the field definitions.

```suggestion
    _concurrent_session_greater_than_zeto = models.Constraint(
        "CHECK(concurent_session IS NULL OR concurent_session_number > 0)",
        "Concurrent session number should be greater than zero.",
    )
```